website/pages/api/packages.ts

```diff
@@ -0,0 +1,59 @@
+import type { NextApiRequest, NextApiResponse } from 'next'
+import { supabase } from '../../lib/supabaseClient'
+import { definitions } from '../../types/database'
+import { apiSuccess, apiNotFound, apiServerError } from '../../lib/helpers'
+
+/**
+ * TYPES
+ *  Note that these aren't strictly accurate as the we haven't "Pick<>"ed the values
+ */
+export type PackageSummary = definitions['packages']
+const summaryFields = `
+    id, slug: package_name, owner: handle, name: partial_name
```

Comment:
> slug: package_name

I found the `package_name`, and `partial_name` a bit confusing and renamed it. I will change this if you prefer to keep this convention though (so that we have a consistent API)

> owner: handle

I also found the `handle` a bit confusing in this context. Perhaps it can just be owner?

Reply:
I'll open a PR against this branch with the updated names so these aliases can be removed